spec/requests/api/v1/docs/display_ads_spec.rb

```diff
@@ -68,6 +68,8 @@
             placement_area: { type: :string, enum: DisplayAd::ALLOWED_PLACEMENT_AREAS,
                               description: "Identifies which area of site layout the ad can appear in" },
             tag_list: { type: :string, description: "Tags on which this ad can be displayed (blank is all/any tags)" },
+            article_exclude_ids: { type: :string,
+                                   description: "Articles this ad should *not* appear (blank means no articles are disallowed, and this ad can appear next to any/all articles). Comma-separated list of integer Article IDs" }, # rubocop:disable Layout/LineLength
```

Comment:
I think its missing a word?

```suggestion
                                   description: "Articles this ad should *not* appear on (blank means no articles are disallowed, and this ad can appear next to any/all articles). Comma-separated list of integer Article IDs" }, # rubocop:disable Layout/LineLength
```

Reply:
I noticed that `article_exclude_ids` were not showing up in the PUT example values. We seem to be missing `type_of` as well too maybe because the CREATE and PUT properties are defined in two separate places. 

<img width="726" alt="Screenshot 2023-03-31 at 11 35 55" src="https://user-images.githubusercontent.com/2786819/229084218-e1be9d8b-740d-4edb-a49c-8a8ce6d8ebd8.png">

If possible, maybe it would  make sense to move it into a schema? Thoughts?

Reply:
Also :eyes: `The ID of the user to unpublish`.